pkg/crd/crdtest/crd_test_util.go

```diff
@@ -35,25 +35,36 @@ func (c *TestCrdClient) CreateCrdWatchers(eventCh chan aadpodid.EventType) (err
 	return nil
 }
 
-func (c *TestCrdClient) RemoveAssignedIdentity(name string) error {
-	delete(c.assignedIDMap, name)
+func (c *TestCrdClient) RemoveAssignedIdentity(assignedIdentity *aadpodid.AzureAssignedIdentity) error {
+	delete(c.assignedIDMap, assignedIdentity.Name)
 	return nil
 }
 
-func (c *TestCrdClient) CreateAssignedIdentity(name string, binding *aadpodid.AzureIdentityBinding, id *aadpodid.AzureIdentity, podName string, podNameSpace string, nodeName string) error {
+// This function is not used currently
```

Comment:
is thic comment relevant ?

Reply:
If it's not being called we should remove. Can always bring it back from history if needed.